packages/vue3/src/layouts/table/Index.vue

```diff
@@ -113,8 +103,12 @@ import { cloneDeep } from "lodash-es";
 import layout from "../../mixins/layout";
 import { key } from "../../utils";
 import { getCurrentInstance } from "vue";
+import Tbody from "./tbody.vue";
```

Comment:
We should not use native HTML tag names as component names. We can prefix it with `The` if there are other such components, we can rename all.